general/press.rst

```diff
@@ -188,13 +188,9 @@ Logos and Graphic material
 :download:`OpenWISP Logo (White Foreground)
 </assets/design/openwisp-logo.svg>`
 
-.. raw:: html
-
-  <div align="center" class="align-center">
-    <img alt="../_downloads/openwisp-logo.svg"
-         src="../_downloads/openwisp-logo.svg"
-         style="background:#000;padding:25px;margin-bottom:24px" />
-  </div>
+.. image:: /assets/design/openwisp.org.svg
```

Comment:
this points to a different logo, moreover, we can't let go of this if we want the white logo to show properly, we need a black background.